homeassistant/components/airthings/sensor.py

```diff
@@ -78,6 +79,13 @@
         translation_key="light",
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    "lux": SensorEntityDescription(
+        key="lux",
+        device_class=SensorDeviceClass.ILLUMINANCE,
+        native_unit_of_measurement=LIGHT_LUX,
+        translation_key="lux",
```

Comment:
Is this line (and the one in `strings.json`) really necessary? A quick search for it and I couldn't find any other integrations using a translation key for this. The reason we have it for light/illuminance (the one above) is that this sensor is not following any standards, like lux. Lux has a `SensorDeviceClass`.

Not 100% sure, but after testing it, it seems like we don't need this. Correct me if I'm wrong